omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js

```diff
@@ -64,39 +65,89 @@ class SessionView extends Component {
 
 
     componentWillMount() {
+
         this.setState(() => {
 
             let sessionData = this.props.courseSessions[this.props.computedMatch.params.course_id][this.props.computedMatch.params.session_id]
             let courseData = this.props.courses[this.props.computedMatch.params.course_id]
-
             let instructorData = this.props.instructors
 
 
-
             return {
                 sessionData: sessionData,
-                courseData: courseData
+                courseData: courseData,
+                instructorData: instructorData
             }
         })
-
-
     }
 
-    render() {
 
-        const flexContainer = {
-            display: "flex",
-            justifyContent: "flex-start",
-            width: "50%",
-            padding: 0,
+
+    // Formatting the date in the view
+    formatDate = (start, end) => {
+        let DayConverter = {
+            1: "Monday",
+            2: "Tuesday",
+            3: "Wednesday",
+            4: "Thursday",
+            5: "Friday",
+            6: "Saturday",
         };
+        let MonthConverter = {
+            0: "January",
+            1: "Febuary",
+            2: "March",
+            3: "April",
+            4: "May",
+            5: "June",
+            6: "July",
+            7: "August",
+            8: "September",
+            9: "October",
+            10: "November",
+            11: "December"
+        }
+
+        let date = new Date(start)
+        let dateNumber = date.getDate()
+        let dayOfWeek = date.getDay()
+        let startMonth = date.getMonth()
+        // Gets days
+        let Days = [dayOfWeek]
+        Days = Days.map((day) => {
+            return DayConverter[day];
+        });
+        //Gets months
+        let Month = [startMonth]
+        Month = Month.map(month => {
+            return MonthConverter[month]
+        })
+        //Start times and end times variable 
+        let startTime = start.slice(11)
+        let endTime = end.slice(11)
+
+        this.timeConverter = (s) => {
```

Comment:
Use more descriptive variable names in this function - in addition to being short, `H` and `h` are easily confusable.